### DISCUSSION TITLE
Student Approved, being reviewed

### DISCUSSION THREAD

**Discussion by DevTDubs** [community/community#114437] [GitHub Education]
My application for student education pack was approved about 30 hours ago, and my benefits have still not been given as it is "still being reviewed"
Is there something I can do about this, or that I should do?![image](https://github.com/community/community/assets/83303005/20fa1f0e-883c-4255-a5a8-23ea8b9cfb0c)

**Comment by GoodGuyPJ** [comment]
Patience: While it's frustrating to experience delays, sometimes these processes take time due to high volumes of applications or internal review procedures. Exercise patience but remain proactive in following up on your application.

**Reply by DevTDubs** [reply]
Oh alright! I was just curious if I needed to do something such as for example verify something. Thank you 😃

**Comment by RkBeau** [comment]
There seems to be quite a few of us stuck in  processing limbo.  Hopefully that may hurry things up.

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.
